test/libsolidity/SemanticTest.cpp

```diff
@@ -78,15 +78,35 @@ TestCase::TestResult SemanticTest::run(ostream& _stream, string const& _linePref
 		for (auto& test: m_tests)
 			test.reset();
 
+		map<string, dev::test::Address> libraries;
+
+		bool constructed = false;
+
 		for (auto& test: m_tests)
 		{
-			if (&test == &m_tests.front())
-				if (test.call().isConstructor)
-					deploy("", test.call().value, test.call().arguments.rawBytes());
+			if (!constructed)
```

Comment:
I think the coding style says that if you use `if-else` the `if` condition should not be negated.

Reply:
Does it? Ok, fine with me :-).